src/register/ConfigurableRegistrationForm.jsx

```diff
@@ -77,6 +78,9 @@ const ConfigurableRegistrationForm = (props) => {
       const { countryCode, displayValue } = countryValidation;
       error = countryValidation.error;
       setFormFields(prevState => ({ ...prevState, country: { countryCode, displayValue } }));
+      if (isRegistrationEmbedded) {
+        return;
```

Comment:
1. why are you adding this at this place of code?
2. why are we restricting it to not run onBlur for country field only? there can be other configurable fields as well and rest of the else blocks are still validation them.